src/applications/discharge-wizard/utilities/display-logic-questions.js

```diff
@@ -12,11 +12,31 @@ export const nextQuestionRoute = (currentQuestion, answer) => {
       if (answer === `${new Date().getFullYear() - 15}`) {
         nextRoute = SHORT_NAME_MAP.DISCHARGE_MONTH;
       } else {
-        nextRoute = SHORT_NAME_MAP.DISCHARGE_REASON;
+        nextRoute = SHORT_NAME_MAP.REASON;
       }
       break;
     case SHORT_NAME_MAP.DISCHARGE_MONTH:
-      nextRoute = SHORT_NAME_MAP.DISCHARGE_REASON;
+      nextRoute = SHORT_NAME_MAP.REASON;
```

Comment:
I noticed you're not using the display conditions (like in PACT Act). It seems to be overcomplicating this routing function. Did the display conditions paradigm not apply well for Discharge Wizard?

Reply:
Largely, display conditions are used in PACT Act to determine what question comes next. Incidentally, we also have separate display conditions for what results page to show. I think DUW is a good match for using them, in particular because it keeps utilities cleaner and gives us a more clear idea of what's coming next.

Reply:
Thanks Randi! I'll go ahead and work on implementing the display condition paradigm and open another PR for those changes when they are ready.

Reply:
@randimays I worked on some changes for the display condition paradigm, I tweaked the solution a bit to work with the single path of questions for DUW. 
https://github.com/department-of-veterans-affairs/vets-website/pull/29130/

I also made some changes in this PR to compare both approaches.

Reply:
I made a comment in the newer PR to this effect, but to reiterate here, since this one is for the existing approach:

This approach doesn't allow us to easily change the order of questions or add/remove questions later on. We can't **assume** those adjustments will be made of course (fully aware that future coding is weird), but since we have a pattern for a flexible routing system defined in PACT Act, it shouldn't take too much bandwidth to implement it for DUW.

Reply:
Sounds good we can use the PACT Act routing system and DCs with the new PR implementation. Thanks for the feedback on both approaches!